src/exception/call_stack.cr

```diff
@@ -22,7 +22,7 @@ struct Exception::CallStack
   # Compute current directory at the beginning so filenames
   # are always shown relative to the *starting* working directory.
   CURRENT_DIR = begin
-    dir = Process::INITIAL_PWD
+    dir = Process::INITIAL_PWD || ""
```

Comment:
This will result with `"/"` - which in turn will make callstack paths erroneously relative.